requirements-test.txt

```diff
@@ -1 +1,2 @@
 coveralls
+openwisp-utils[qa]>=0.3.1
```

Comment:
Lol, this might be a little frustrating but 3.2 got released, can you please update again? :joy: